apps/server/src/api-data/excel/parsing/utils.ts

```diff
@@ -0,0 +1,169 @@
+import { CustomFieldKey, CustomFields, MaybeNumber } from 'ontime-types';
```

Comment:
question: how about naming this excel.utils.ts and leaving it at the root directory with the other excel things? That is at least the pattern we have from the other parts of the code

Reply:
can do